package.json

```diff
@@ -59,7 +59,7 @@
   "scripts": {
     "_comment": "ONLY RUN BUILD IF YOUR SYSTEM HAS 8GB OR MORE OF RAM",
     "build": "node seed && node --max_old_space_size=7168 ./node_modules/.bin/gatsby build",
-    "dev": "node seed && node --max_old_space_size=7168 ./node_modules/.bin/gatsby develop",
+    "dev": "gatsby develop",
```

Comment:
We would still need to run `node seed` before running gatsby. The seed script builds a JSON which is used to create the side navigation.